core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/pythonworker/PythonWorkflowWorker.scala

```diff
@@ -2,6 +2,7 @@ package edu.uci.ics.amber.engine.architecture.pythonworker
 
 import akka.actor.{ActorRef, Props}
 import com.typesafe.config.{Config, ConfigFactory}
+import edu.uci.ics.amber.engine.architecture.pythonworker.PythonWorkflowWorker.getFreeLocalPort
```

Comment:
is this import necessary? also please avoid these kind of formatting only changes. if possible, please revert this file.

Reply:
I moved the `getFreeLocalPort` to the `object PythonWorkflowWorker` as a static function. So this is not just a formatting only change. I can revert the change and include it in future PRs.